package.json

```diff
@@ -8,9 +8,13 @@
     "start": "next start",
     "lint": "next lint",
     "format": "prettier --check --ignore-path .gitignore .",
-    "format:fix": "prettier --write --ignore-path .gitignore ."
+    "format:fix": "prettier --write --ignore-path .gitignore .",
+    "seed:tasks": "tsx --tsconfig ./tsconfig.scripts.json ./src/mocks/seed-visitors.ts"
   },
   "dependencies": {
+    "@radix-ui/react-checkbox": "^1.0.4",
```

Comment:
It is a good practice to have exact versions fro the packages defined, i.e. 

```diff
-    "@radix-ui/react-checkbox": "^1.0.4",
+    "@radix-ui/react-checkbox": "1.0.4"
```

It will help with debugging and minify error because of inconsistencies between minor versions of a package